.github/actions/contentful-teardown/action.yml

```
@@ -0,0 +1,40 @@
+name: Contentful Teardown
+description: Tears down Contentful Environment if required
+
+inputs:
+  contentful-space-id:
+    description: 'Contentful Space ID'
+    required: true
+  contentful-token:
+    description: 'Contentful access token'
+    required: true
+  contentful-env:
+    description: 'Contentful environment'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Remove Contentful PR Environment
+      if: ${{ inputs.contentful-env != 'Development' }} && ${{ inputs.contentful-env != 'Production' }}
+      shell: bash
+      run: |
+        STATUS_CODE=$(curl --silent \
+                           --request DELETE \
+                           --output /dev/null \
+                           --write-out "%{http_code}" \
+                           --header "Authorization: Bearer ${CONTENTFUL_MANAGEMENT_ACCESS_TOKEN}" \
+                           https://api.contentful.com/spaces/${CONTENTFUL_SPACE_ID}/environments/${CONTENTFUL_ENV_ID})
+
+        if [ "$STATUS_CODE" == "204" ]; then
```

Should this be:

```suggestion
        if [ "$STATUS_CODE" = "204" ]; then
```
or
```suggestion
        if [[ "$STATUS_CODE" == "204" ]]; then
```